data/actions/lib/actions.lua

```diff
@@ -199,6 +199,7 @@ function onUseShovel(player, item, fromPosition, target, toPosition, isHotkey)
 
 		toPosition.z = toPosition.z + 1
 		tile:relocateTo(toPosition)
```

Comment:
Not related to this PR, but I've just checked how it behaves in Tibia and...

![Screenshot 2021-06-21 at 14 45 49](https://user-images.githubusercontent.com/57528542/122764165-789b2a80-d29f-11eb-8a10-e7d58ffc8cdb.png)

Item(s) blocking loose stone pile is not relocated 🤔.
I'd see what happens with players, but I have no friends 😢

Reply:
![Screenshot 2021-06-21 at 14 54 41](https://user-images.githubusercontent.com/57528542/122765432-c95f5300-d2a0-11eb-8af7-1635b51f13a9.png)

Fields are also not removed. They are removed `onStepIn`. Relocation happens `onStepIn` too. 🤔

Reply:
🤔

Reply:
also desert quest entrance stone pile doesn't make player go down on opening and I have no idea why

Reply:
The sand stone pile also relocates the items when using shovel. #cip quality 😎 
Anyway, the auto-enter when using shovel was introduced around 2016, knowing cip they just forgot to update every item to act the same 🤷